torch/onnx/symbolic_opset9.py

```diff
@@ -2165,9 +2166,11 @@ def batch_norm(
         return res
 
 
-@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
-def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
-    if symbolic_helper.is_caffe2_aten_fallback():
+def _layer_norm_shared(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
+    if (
```

Comment:
The condition changed. Do you know if this is due to outdated code merge, or is it intended?

Reply:
It's by accident and will be reverted. Thanks.